webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -4858,6 +4860,22 @@ var KM_ATTACHMENT_V2_SUPPORTED_MIME_TYPES = ["application","text","image"];
                 $applozic.template("searchContactbox", searchContactbox);
                 $applozic.template("csatModule", csatModule);
             };
+            _this.loadDropdownOptions = function () {
+                // Mid conversation CSAT
+                // update if dedicated parameter is introduced
+                var CSATTrigger = document.getElementById("km-csat-trigger");
+                !(kommunicate &&
+                    kommunicate._globals &&
+                    kommunicate._globals.collectFeedback) &&
```

Comment:
@Prochnost55 use below variable everywhere you're accessing collectFeedback like above.
CSAT_ENABLED

NOTE: Inside below function also you need to update the same.
```
_this.getGroup = function (params) {
```